language.py

```diff
@@ -1498,6 +1498,8 @@ def to_lean_binop(bop, state):
   bitwidth = unify_bitwidths([lv1.expr.bw(), lv2.expr.bw()])
   propagate_bitwidth(lv1.expr, bitwidth)
   propagate_bitwidth(lv2.expr, bitwidth)
+  if bop.flags != []:
+    raise RuntimeError("binop with flags not supported")
```

Comment:
```suggestion
    raise RuntimeError("binops with flags '%s' not supported" % (bop.flags))
```

This should help us incrementally add more flags, so the error message that is logged in `Alive.csv` can tell us what flags are missing.